src/sections/Contact/contact-content.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+
+// Images
+import Jobs_White from "../../images/contact/jobs_white.svg";
+import Support_White from "../../images/contact/support_white.svg";
+import Partner_White from "../../images/contact/partner_white.svg";
+import Contact_White from "../../images/contact/contact_white.svg";
+
+import ContactCard from "../../components/ContactCard/ContactCard";
+import { Container, Col, Row} from "../../reusecore/Layout";
+import ContactCardStyle from "./contactcard.style";
+
+
+const ContactContent = () => {
+  return (
+  <ContactCardStyle>
+    <Container>
+      <Row Hcenter>
+        <Col xs={12} sm={12} md={6} lg={6} xl={6}>
```

Comment:
The 3 cards in a row was a better configuration for md devices and as Lee stated you will require space in the second row to incorporate the graphic. So please change this to 3 cards in a row.

Reply:
Ok sure will do that right away.